src/agent/debuglet.ts

```diff
@@ -205,6 +205,15 @@ export class Debuglet extends EventEmitter {
    * @private
    */
   async start(): Promise<void> {
+    process.on('warning', (warning) => {
+      if ((warning as any).code ===
+          'INSPECTOR_ASYNC_STACK_TRACES_NOT_AVAILABLE') {
+        console.log(
+            'The current debug agent does not use Inspector async stack ' +
+            'traces. The above warning will not affect the debug agent.');
```

Comment:
Do we know for sure that the warning message will be printed before the `console.log` message?  If not, the use of `The above warning` wouldn't be correct.

Reply:
Since warning has been listened, once we reach this 'console.log' code, the warning was already fired and the message was printed. To be more concise, I changed the message to 'The Stackdriver debug agent does not use Inspector async stack traces. The INSPECTOR_ASYNC_STACK_TRACES_NOT_AVAILABLE warning does not impact Stackdriver debug agent and can be ignored.'.